template/build.gradle.kts

```diff
@@ -0,0 +1,57 @@
+plugins {
+    kotlin("multiplatform")
+    id("jacoco")
+    id("maven-publish")
+    id("com.android.library")
+    id("org.jlleitschuh.gradle.ktlint")
+}
+
+val ext = (gradle as ExtensionAware).extra
+
+apply(from = "../gradle/publishable_component.gradle")
+
+group = "com.splendo.kaluga"
+version = ext["library_version"]!!
+
+repositories {
+    google()
+    mavenCentral()
+}
+
+dependencies {
+    implementation("androidx.fragment:fragment:1.1.0")
+    implementation("androidx.appcompat:appcompat:1.1.0")
+    testImplementation("org.mockito:mockito-core:2.28.2")
+    testImplementation("org.junit.jupiter:junit-jupiter-api:5.3.1")
+    testRuntimeOnly("org.junit.jupiter:junit-jupiter-engine:5.3.1")
+    androidTestImplementation("androidx.test.uiautomator:uiautomator:2.2.0")
+    androidTestImplementation("androidx.fragment:fragment-ktx:1.1.0")
```

Comment:
Version numbers are hardcoded too.

Reply:
Everything was copypaste from HUD and Alerts. I'll check all submodules and make changes if needed

Reply:
you were looking at (quite) old versions of those, as you can see in master they don't contain many of the things you put in (because they are in `component.gradle` or obsolete).